task_opts.go

```diff
@@ -49,7 +49,7 @@ func WithRootFS(mounts []mount.Mount) NewTaskOpts {
 // instead of resolving it from runtime name.
 func WithRuntimePath(absRuntimePath string) NewTaskOpts {
 	return func(ctx context.Context, client *Client, info *TaskInfo) error {
-		info.runtime = absRuntimePath
+		info.RuntimePath = absRuntimePath
```

Comment:
Is this a breaking change?

Reply:
This is not a blocking modification

In the past if the user used WithRuntimePath alone then the runtime path would not take effect.

And if the user uses WithRuntimePath + other options (e.g. WithNoPivotRoot), then the wrong runtime options are generated and the task creation fails

`WithNoPivotRoot` and other TaskOpts generate the runtime options based on runtime.
For `io.containerd.runc.*` runtime, if runtime changes to runtime path, the generated option type is not `*options.Options`